components/script/dom/globalscope.rs

```diff
@@ -711,6 +711,16 @@ impl GlobalScope {
         unreachable!();
     }
 
+    pub fn runtime_handle(&self) -> ParentRuntime {
+        if self.is::<Window>() {
+            ScriptThread::runtime_handle()
+        } else if let Some(worker) = self.downcast::<WorkerGlobalScope>() {
+            worker.runtime_handle()
+        } else {
```

Comment:
I do not believe there is any way to create a worker from a worklet, so there would be no need to implement this for it.

Reply:
Sorry, I didn't say it clearly.

IIUC, the `runtime_handle` is used to avoid new threads created before the old threads deleted.
So I'd like to ask, should `WorkletGlobalScope` also handle the `runtime_handle` 👀?

```suggestion
        } else if let Some(worklet) = self.downcast::<WorkletGlobalScope {
            worklet.runtime_handle()
        } else {
```

Reply:
It would only matter if we were trying to make a worker inside of a worklet, since this code runs on the caller global scope.

Reply:
Understood! Thanks!